doc/user/sql/types/float.md

```diff
@@ -22,7 +22,29 @@ Type               | Aliases           | Size    | Range
 
 - Materialize assumes untyped numeric literals containing decimal points are [`decimal`](../decimal); to use `float`, you must explicitly cast them as we've done below.
 
-## Details
```

Comment:
Just for consistency with other docs, can you leave this h2 in place?

Reply:
There are two `## Details` headers in this particular file. Seemed like the first one was the right one to keep?

Reply:
Ah, you're right! I just looked at the diff and saw it got yanked. Apologies.